### PR TITLE
increase test coverage of `cairo-native-run`

### PR DESCRIPTION
**Pull request by tcoratger**
## Checklist
- [ ] Linked to Github Issue
- [ ] Unit tests added
- [ ] Integration tests added.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.

### CODE REVIEW

**Comment by tcoratger** [comment]
@edg-l Concerning `check_compiler_path`, shouldn't we check also that, if the path leads to a file, the extension of this file should be `.cairo`?

**Review by edg-l — DISMISSED** [review]

File: src/bin/cairo-native-run.rs [hunk 1]
```diff
@@ -268,16 +257,15 @@ fn jitvalue_to_felt(value: &JitValue) -> Vec<Felt> {
                 if debug_name == "core::bool" {
                     vec![(*tag == 1).into()]
                 } else {
-                    felts.push((*tag).into());
-                    let felt = jitvalue_to_felt(value);
-                    felts.extend(felt);
+                    let mut felts = vec![(*tag).into()];
+                    // felts.push((*tag).into());
```
Comment (juanbono) [inline src/bin/cairo-native-run.rs:261]: Why did you comment that line?

**Review by juanbono — APPROVED** [review]

**Review by juanbono — CHANGES_REQUESTED** [review]
Check comment

File: src/bin/cairo-native-run.rs (hunk 1 above)
Comment (tcoratger) [inline src/bin/cairo-native-run.rs:261]:
```suggestion
```

Comment (tcoratger) [reply src/bin/cairo-native-run.rs]: Sorry, we can just remove it as the vec is initialized on the line before with the tag. Just fixed

**Review by edg-l — APPROVED** [review]

**Pull request merged**